neovim/plugin/host.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, nvim):
             'shutdown': self.shutdown
         }
         self._nvim_encoding = nvim.options['encoding']
+        if IS_PYTHON3:
+            self._nvim_encoding = self._nvim_encoding.decode('ascii')
```

Comment:
Isn't it possible for `self._nvim_encoding` to already be a string(in which case the "decode" will fail)